src/components/MultipleAvatars.js

```diff
@@ -44,50 +49,65 @@ const MultipleAvatars = (props) => {
     }
 
     if (props.avatarImageURLs.length === 1) {
+        const tooltipToPass = _.isArray(props.tooltip) ? props.tooltip.join(', ') : props.tooltip;
         return (
             <View style={avatarContainerStyles}>
-                <Avatar
-                    source={props.avatarImageURLs[0]}
-                    size={props.size}
-                    isChatRoom={props.isChatRoom}
-                    isArchivedRoom={props.isArchivedRoom}
-                />
+                <Tooltip text={tooltipToPass}>
+                    <Avatar
+                        source={props.avatarImageURLs[0]}
+                        size={props.size}
+                        isChatRoom={props.isChatRoom}
+                        isArchivedRoom={props.isArchivedRoom}
+                    />
+                </Tooltip>
             </View>
         );
     }
 
+    const tooltip = props.tooltip;
+    const tooltipForFirstAvatar = _.isArray(tooltip) ? tooltip[0] : tooltip;
+    const tooltipForAvatarCount = _.isArray(tooltip) && tooltip.length > 1 ? tooltip.slice(1).join(', ') : '';
```

Comment:
I think a short comment here might be useful for readability.

Reply:
I guess with the new comments, this won't be required now.